src/native/managed/cdac/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/ComWrappers_1.cs

```diff
@@ -22,4 +22,72 @@ public TargetPointer GetComWrappersIdentity(TargetPointer address)
         Data.NativeObjectWrapperObject wrapper = _target.ProcessedData.GetOrAdd<Data.NativeObjectWrapperObject>(address);
         return wrapper.ExternalComObject;
     }
+
+    private bool GetComWrappersCCWVTableQIAddress(TargetPointer ccw, out TargetPointer vtable, out TargetPointer qiAddress)
+    {
+        vtable = TargetPointer.Null;
+        qiAddress = TargetPointer.Null;
+        try
+        {
+            TargetPointer vtableAddress = _target.ReadPointer(ccw);
+            if (vtableAddress == TargetPointer.Null)
+                return false;
+            qiAddress = _target.ReadPointer(vtableAddress);
```

Comment:
This makes it so we don't need to call `.Value` below.
```suggestion
            qiAddress = _target.ReadCodePointer(vtableAddress);
```

Reply:
As qiAddress is a TargetPointer, this would mean I need an intermediate variable other than qiAddress. I am not sure that this would be any cleaner, and it would not match the DAC code.